test/evergreen.yml

```diff
@@ -28,6 +29,39 @@ exec_timeout_secs: 21600 # 6 hrs
 #######################################
 
 functions:
+  "setup environment":
+    - command: expansions.update
+      params:
+        updates:
+        # The expansion is used for each task that runs a WiredTiger test. All of these variables
+        # are common among the build variants, if there are any specific variables that needs to be
+        # set, users can add onto the additional_env_vars in the variant
+        - key: PREPARE_TEST_ENV
+          value: |
+            export WT_TOPDIR=$(git rev-parse --show-toplevel)
+            export WT_BUILDDIR=$WT_TOPDIR/cmake_build
+
+            if [ "$OS" = "Windows_NT" ]; then
+              export PATH=/cygdrive/c/python/Python311:/cygdrive/c/python/Python311/Scripts:$PATH
+              export PYTHONPATH="($WT_TOPDIR/lang/python/wiredtiger):$(cygpath -w $WT_TOPDIR/lang/python)"
+            else
+              export PATH=/opt/mongodbtoolchain/v4/bin:$PATH
```

Comment:
Please note neither Windows nor macOS has toolchain support. I don't expect any issue with setting PATH for their builders, but may be worth paying special attention during testing to make sure no surprises.